main/ruby/APKBUILD

```diff
@@ -94,7 +94,9 @@ package() {
 	rm -R "$pkgdir"$_gemdir/cache/*
 
 	# Remove bundled CA certificates.
-	rm -R "$pkgdir"$_rubydir/rubygems/ssl_certs/*.pem
+	if [ -e "$pkgdir"$_rubydir/rubygems/ssl_certs/*.pem ]; then
+		rm -R "$pkgdir"$_rubydir/rubygems/ssl_certs/*.pem
+	fi
```

Comment:
This condition is unnecessary and also wrong. It’s enough to just add `|| true` after the command that may fail, i.e. `rm -R "$pkgdir"$_rubydir/rubygems/ssl_certs/*.pem || true`.